src/components/ModalHeader/ModalHeader.tsx

```diff
@@ -0,0 +1,44 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './ModalHeader.module.css';
+
+export type Props = {
+  /**
+   * Child node(s) that can be nested inside component. `ModalHeader`, `ModalBody`, and `ModelFooter` are the only permissible children of the Modal
```

Comment:
This comment I pointed out before is on the `ModalHeader` too btw.

Reply:
God dang it thanks for all the catches